Client/Api/Commands/IModifyProcessInstanceCommandStep1.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using GatewayProtocol;
+using Zeebe.Client.Api.Responses;
+
+namespace Zeebe.Client.Api.Commands;
+
+public interface IModifyProcessInstanceCommandStep1 :
+    IFinalCommandWithRetryStep<IModifyProcessInstanceResponse>
+{
```

Comment:
🤔 Is this useful to have none, modification? I feel not. Lets introduce at least another step.

Reply:
I didn't understand here, please explain in more detail.